### PR TITLE
Finish implementing C++ se2-car.cpp example

### PR DESCRIPTION
**Pull request by ManifoldFR**
This PR:

* finishes the se2-car.cpp example
* fixes the Doxyfile's python directory

### CODE REVIEW

File: examples/se2-car.cpp [hunk 1]
```diff
@@ -5,66 +5,99 @@
...
+using SE2 = proxsuite::nlp::SETpl<2, T>;
 
 using namespace aligator;
 using StateError = StateErrorResidualTpl<T>;
 using QuadResidualCost = QuadraticResidualCostTpl<T>;
+using context::StageModel;
+using context::TrajOptProblem;
+ALIGATOR_DYNAMIC_TYPEDEFS(T);
+
+struct CarDynamics : dynamics::ODEAbstractTpl<T> {
```
Comment (jorisv) [inline examples/se2-car.cpp:24]: You can write the function in latex to help people understand the model

Comment (jorisv) [reply examples/se2-car.cpp:24]: If you're motivated, you can also write the Ju, Jx equations (this will help to understand the matrix dimensions)

Comment (ManifoldFR) [reply examples/se2-car.cpp:32]: Done! :)

**Review by jorisv — APPROVED** [review]

**Review by jcarpent — APPROVED** [review]
Well done ! ;-)

**Pull request merged**